code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -267,7 +267,7 @@ GLOBAL_LIST_INIT(titaniumglass_recipes, list(
 
 /obj/item/stack/sheet/titaniumglass/examine_more(mob/user)
 	. = ..()
-	. += "Titanium-silicate mixes are an old but highly effective technology that produce a relatively lightweight, very strong glass that can withstand a good amount of punishment."
+	. += "Titanium-silica mixes are an old but highly effective technology that produce a relatively lightweight, very strong glass that can withstand a good amount of punishment."
```

Comment:
Forgot to update this one

Reply:
This one is fine due to the sentence structure